src/Components/userForm.js

```diff
@@ -0,0 +1,39 @@
+export const Userform = (props) => {
+  return (
+    <form>
+      <div className="formControl">
```

Comment:
Suggestion: you can create one component named **FormInput** and pass data through props, rather than rewriting the same code.

Reply:
ok got it